<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFrameClass</class>
 <widget class="QMainWindow" name="MainFrameClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CvsStat</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New_Stat"/>
    <addaction name="action_Update_Stat"/>
    <addaction name="action_Remove_Stat"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Query"/>
    <addaction name="separator"/>
    <addaction name="action_View"/>
    <addaction name="action_View_Diff"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Export"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Tools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New_Stat">
   <property name="text">
    <string>&amp;New Stat...</string>
   </property>
   <property name="statusTip">
    <string>create a new cvs respoiste stat</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Query">
   <property name="text">
    <string>&amp;Query</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_Update_Stat">
   <property name="text">
    <string>Update Stat</string>
   </property>
  </action>
  <action name="action_Remove_Stat">
   <property name="text">
    <string>&amp;Remove Stat</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_View_Diff">
   <property name="text">
    <string>&amp;View Diff...</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="action_View">
   <property name="text">
    <string>&amp;View</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="action_Export">
   <property name="text">
    <string>&amp;Export...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
